### PR TITLE
feat(view): #MC-165 init external calendar import view

### PR DESCRIPTION
**Pull request by alicedraillard**
## Describe your changes
-add "add external calendar" button
-add "external calendar" section in sidebar with new component for items

## Checklist tests
see if formentionned items appear
test sidebar new section :
- empty state if there are no calendars
- try to (un)select calendar
- try ton (un)check calendar checkboxes and use menu buttons (edit, ...)

## Issue ticket number and link
#MC-165 https://jira.support-ent.fr/browse/MC-165

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/ts/directives/side-bar/side-bar.ts [hunk 1]
```diff
@@ -84,7 +86,7 @@ export const sideBar = ng.directive('sideBar', () =>{
             }
 
             vm.isMyCalendar = (calendar: Calendar) : boolean => {
-                return calendar.owner.userId == $scope.$parent.me.userId;
+                return (calendar.owner.userId == $scope.$parent.me.userId) && !vm.isExternalCalendar(calendar);
```
Comment (alicedraillard) [inline src/main/resources/public/ts/directives/side-bar/side-bar.ts:89]: filter out external calendars from calendar list for the 'my calendars' section

File: src/main/resources/public/template/top-menu.html [hunk 2]
```diff
@@ -15,5 +15,8 @@ <h1>
     <div class="right-magnet">
       <button workflow="calendar.admin" ng-click="newCalendar()"><i18n>calendar.new</i18n></button>
     </div>
+    <div class="right-magnet">
+      <button workflow="calendar.admin"><i18n>calendar.add.external.calendar</i18n></button>
+    </div>
```
Comment (BastienBenard) [inline src/main/resources/public/template/top-menu.html:20]: Ce sera utilisé plus tard ?

**Review by BastienBenard — APPROVED** [review]

File: src/main/resources/public/template/top-menu.html (hunk 2 above)
Comment (alicedraillard) [reply src/main/resources/public/template/top-menu.html:20]: oui

Comment (JinDouang) [reply src/main/resources/public/template/top-menu.html:20]: A terme ça sera un composant qui intègrera la logique des agendas externes

**Review by JinDouang — APPROVED** [review]

**Pull request merged**